tests/test_pipenv.py

```diff
@@ -294,6 +294,24 @@ def test_install_without_dev(self, pypi):
             c = p.pipenv('run python -c "import tablib"')
             assert c.return_code == 0
 
+    @pytest.mark.cli
+    @pytest.mark.install
+    def test_install_without_dev_section(self, pypi):
+        with PipenvInstance(pypi=pypi) as p:
+            with open(p.pipfile_path, 'w') as f:
+                contents = """
```

Comment:
I've got a little tip for you, apologies if you're already aware of it.

You can use `textwrap.dedent` to make this read more nicely in the source code:

```
                     contents = dedent("""
                          [packages]
                          tablib = "*"
                     """)
```